browser/brave_wallet/BUILD.gn

```diff
@@ -3,21 +3,46 @@ import("//extensions/buildflags/buildflags.gni")
 
 assert(brave_wallet_enabled)
 
-source_set("brave_wallet") {
+source_set("infobar") {
+  visibility = [
+    "//brave/browser/*",
+    "//brave/test:brave_browser_tests",
+    "//chrome/browser/*",
+  ]
+
+  sources = [
+    "crypto_wallets_infobar_delegate.h",
+  ]
+
+  deps = [
+    "//base",
+    "//components/infobars/core",
+    "//url",
+  ]
+}
+
+source_set("brave_wallet_impl") {
+  visibility = [ "//chrome/browser" ]
```

Comment:
this is intentionally `//chrome/browser` and not `//chrome/browser/*` so it can't be added as a direct dep anywhere else